### PR TITLE
Add scheme for WS - Correct Dataset

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: code_schemes/ws_correct_dataset.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
+{
+  "SchemeID": "de6bfc6e",
+  "Name": "WS - Correct Dataset",
```
Comment (lukechurch) [inline code_schemes/ws_correct_dataset.json:3]: This is interesting - it implies that 'WS' is actually wrong, it should be WD - as data is often coded against multiple schemes within a single dataset.

**Comment by ParthaMoman** [comment]
can someone clarify what the purpose of this code is?

**Comment by as2388** [comment]
> can someone clarify what the purpose of this code is?

For any message which is coded as “WS”, a separate code should be assigned in this scheme so that we know which dataset that message should be moved to.

**Comment by ParthaMoman** [comment]
hmmm, I am not sure if they match the actual variable names? presumably this is required?

File: code_schemes/ws_correct_dataset.json (hunk 1 above)
Comment (as2388) [reply code_schemes/ws_correct_dataset.json:3]: I think you're right. Should we think about changing this now or wait until the next project?

**Review by lukechurch — APPROVED** [review]

**Comment by lukechurch** [comment]
> hmmm, I am not sure if they match the actual variable names? presumably this is required?

We general use lookup tables to map between the two

**Pull request merged**